src/ramanchada2/protocols/calibration.py

```diff
@@ -582,20 +614,22 @@ def __init__(self, laser_wl: int):
         Example:
         ```python
         # Create an instance of CalibrationModel
-        calmodel = CalibrationModel(laser_wl=785)
-        calmodel.derive_model_x(
+        import ramanchada2 as rc2
+        import ramanchada2.misc.constants as rc2const
+        from ramanchada2.protocols.calibration import CalibrationModel
```

Comment:
Why do we put these imports here, and why do we import the whole rc2?

Reply:
this is example in pydoc

Reply:
Ugh, my bad, thanks!